website/_dev/index.cn.md

```diff
@@ -16,8 +16,10 @@ Commits [总结](https://github.com/apache/kylin/commits/master)
 Gitbox 的镜像：[https://gitbox.apache.org/repos/asf?p=kylin.git ](https://gitbox.apache.org/repos/asf?p=kylin.git )
 
 ### CI 和代码分析
-使用 JDK 1.8 的 master 分支上的 UT：[Kylin-Master-JDK-1.8](https://builds.apache.org/job/Kylin-Master-JDK-1.8/) 
-静态代码分析：[SonarCube dashboard](https://builds.apache.org/analysis/overview?id=org.apache.kylin%3Akylin)
+使用[GitHub Actions](https://github.com/apache/kylin/actions) 执行Linux x86平台上的单元测试，使用
```

Comment:
Hi @martin-g , I think both x86 and arm64 run on  `https://app.travis-ci.com/github/apache/kylin`, but they run separately in two times.

Reply:
I've updated the English version!